components/admin/users/AdminUsersTable.tsx

```diff
@@ -56,9 +57,9 @@ const AdminOption: React.FC<AdminOptionProps> = ({
     }
   }
 
-  const changeButton = async () => {
-    await changeRights(mutationVariable)
-    const newUsers = [...users]
+  const changeButton = () => {
+    changeRights(mutationVariable)
+    const newUsers = _.cloneDeep(users)
```

Comment:
Was there a problem that was occurring? Why is the spread operator replaced with `._cloneDeep` ?